### PR TITLE
[no ticket] pure formatting change

### PR DESCRIPTION
**Pull request by Qi77Qi**
Pure formatting change. Took the config from https://github.com/typelevel/cats-effect/blob/master/.scalafmt.conf

Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/Leonardo.scala [hunk 1]
```diff
@@ -60,10 +60,9 @@ object Leonardo extends RestClient with LazyLogging {
       handleClusterSeqResponse(parseResponse(getRequest(s"$url/$path")))
     }
 
-    def create(googleProject: GoogleProject,
-               clusterName: ClusterName,
-               clusterRequest: ClusterRequest)
-              (implicit token: AuthToken): Cluster = {
+    def create(googleProject: GoogleProject, clusterName: ClusterName, clusterRequest: ClusterRequest)(
+      implicit token: AuthToken
+    ): Cluster = {
```
Comment (kyuksel) [inline automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/Leonardo.scala:65]: I wonder if there's a way to make it format input arg lists like this instead:

```scala
def create(googleProject: GoogleProject, clusterName: ClusterName, clusterRequest: ClusterRequest)
          (implicit token: AuthToken): Cluster = {
```

File: automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/lab/LabNotebookPage.scala [hunk 2]
```diff
@@ -16,67 +16,82 @@ import scala.concurrent.duration._
...
-  }
 
   // menu elements
 
   lazy val fileMenu: Element = {
-    findAll(cssSelector(menus)).filter { e => e.text == "File" }.toList.head
+    findAll(cssSelector(menus))
+      .filter { e =>
+        e.text == "File"
+      }
```
Comment (kyuksel) [inline automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/lab/LabNotebookPage.scala:36]: Splitting this over 3 lines is overkill but I'm guessing the formatter can't be smart enough to figure out that the functional literal is pretty short.

File: .scalafmt.conf [hunk 3]
```diff
@@ -0,0 +1,10 @@
+version = 2.0.1
+align = none
+align.openParenCallSite = true
+align.openParenDefnSite = true
+maxColumn = 120
+continuationIndent.defnSite = 2
+assumeStandardLibraryStripMargin = true
+danglingParentheses = true
+rewrite.rules = [AvoidInfix, SortImports, RedundantBraces, RedundantParens, SortModifiers]
+docstrings = JavaDoc
```
Comment (kyuksel) [inline .scalafmt.conf:10]:
```suggestion
docstrings = JavaDoc

```

File: .scalafmt.conf [hunk 4]
```diff
@@ -0,0 +1,10 @@
+version = 2.0.1
+align = none
+align.openParenCallSite = true
+align.openParenDefnSite = true
+maxColumn = 120
+continuationIndent.defnSite = 2
+assumeStandardLibraryStripMargin = true
+danglingParentheses = true
+rewrite.rules = [AvoidInfix, SortImports, RedundantBraces, RedundantParens, SortModifiers]
```
Comment (kyuksel) [inline .scalafmt.conf]: Should we use `ScalaDoc` instead?

File: .scalafmt.conf [hunk 5]
```diff
@@ -0,0 +1,10 @@
+version = 2.0.1
+align = none
+align.openParenCallSite = true
+align.openParenDefnSite = true
+maxColumn = 120
```
Comment (kyuksel) [inline .scalafmt.conf:5]: I see lines spanning at least 146 chars here and there for some reason but most are under 120, which is a dream come true for me  😇 .

**Review by kyuksel — APPROVED** [review]
Looks great to me overall. Thanks for taking the time to put this in. There are a few formattings that aren't ideal to me (some of which I pointed out inline) but overall it looks much better and consistent. It also keeps most of the lines under 120 chars which allows me to have two vertically-split panes on my IntelliJ per monitor without scrolling horizontally, which is a dream come true  😄

**Comment by kyuksel** [comment]
Could you document (on README?) how to run the formatter ourselves, and perhaps add it to the PR checklist?

**Comment by Qi77Qi** [comment]
@kyuksel  I think we can further tune the configuration over time, so I'm going to merge this once tests pass

**Comment by rtitle** [comment]
Works for me - will Intellij automatically pick this up, or any instructions on how to import the formatting changes to Intellij?

**Review by rtitle — APPROVED** [review]

**Comment by Qi77Qi** [comment]
@rtitle yea, see https://scalameta.org/scalafmt/docs/installation.html#format-on-save...this page is linked in readme. So people can choose what they want to set up locally

**Comment by Qi77Qi** [comment]
jenkins retest

**Pull request merged**